solr/solr-ref-guide/src/cluster-node-management.adoc

```diff
@@ -420,7 +632,29 @@ http://localhost:8983/solr/admin/collections?action=REMOVEROLE&role=overseer&nod
 
 Returns the current status of the overseer, performance statistics of various overseer APIs, and the last 10 failures per operation type.
 
-`/admin/collections?action=OVERSEERSTATUS`
+[.dynamic-tabs]
+--
+[example.tab-pane#v1overseerstatus]
+====
+[.tab-label]*V1 API*
+
+[source,bash]
+----
+http://localhost:8983/solr/admin/collections?action=OVERSEERSTATUS
+
+----
+====
+
+[example.tab-pane#v2overseerstatus]
+====
+[.tab-label]*V2 API*
+
+[source,bash]
+----
+curl -X GET http://localhost:8983/api/cluster/overseer -H 'Content-Type: application/json'
```

Comment:
[0] GET requests don't need a `Content-type` header afaik.  It doesn't do any harm, but might be simpler to leave it off 🤷

Reply:
Makes sense, and we don't do it anywhere else!